### ISSUE TITLE
Cannot show node statistics in high-load situations

### ISSUE THREAD

**Issue by uplexPaul** [Graylog2/graylog2-server#5789]
I've just updated to the new graylog 3.0.0.

Then i was configuring the new graylog-sidecar on some sending servers. I did accidentally configured some of the filebeat inputs to "tail: false" which led to that very big log files with hundreds of thousands of messages were read in again.

I then wanted to check the node statistics via the graylog-webinterface. On the page "/system/nodes" i could sometimes see the current usage information but most of the times it stated that the information is currently not available. When i then clicked on the node and landed on "/system/nodes/<<UUID>>" that page shortly showed the correct usage information and then the page switched to an "ooops" page showing the following error:

```
TypeError: Cannot read property 'org.graylog2.buffers.input.usage' of undefined
    at b.render (https://graylog.umg-cms.eu/assets/27059c95-10.567b98aaba093175141f.js:1:10755)
    at ct (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:18:59598)
    at zt (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:18:59393)
    at it (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:18:63066)
    at Vt (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:18:78129)
    at Ut (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:18:78506)
    at Tn (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:18:84899)
    at gn (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:18:84286)
    at Rn (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:18:83307)
    at $t (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:18:82227)
    at Object.enqueueSetState (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:18:47300)
    at b.W.setState (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:26:1660)
    at M.e (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:10:1321)
    at M.emit (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:82:1058853)
    at n.trigger (https://graylog.umg-cms.eu/assets/vendor.4024e2a8db732781a971.js:10:2196)
    at https://graylog.umg-cms.eu/assets/builtins.567b98aaba093175141f.js:158:109610
    at a (https://graylog.umg-cms.eu/assets/builtins.567b98aaba093175141f.js:158:88318)
    at A._settlePromiseFromHandler (https://graylog.umg-cms.eu/assets/builtins.567b98aaba093175141f.js:158:61004)
    at A._settlePromise (https://graylog.umg-cms.eu/assets/builtins.567b98aaba093175141f.js:158:61887)
    at A._settlePromise0 (https://graylog.umg-cms.eu/assets/builtins.567b98aaba093175141f.js:158:62627)
    at A._settlePromises (https://graylog.umg-cms.eu/assets/builtins.567b98aaba093175141f.js:158:63970)
    at https://graylog.umg-cms.eu/assets/builtins.567b98aaba093175141f.js:158:16848

Component Stack:

    in BufferUsage
    in div
    in p
    in div
    in p
    in div
    in p
    in div
    in p
    in div
    in t
    in div
    in i
    in ShowNodePage
    in n
    in div
    in p
    in div
    in p
    in div
    in t
    in div
    in t
    in n
    in withRouter(n)
    in div
    in App
    in RouterContext
    in Router
    in h
    in t
    in n
    in AppFacade
```
All buffers were full and i had like ~750k messages in the disk journal using 7 segments in total.

## Expected Behavior
Node usage data should be always available.
Detailed page should not show an oops page when information about the node is currently not available.

## Current Behavior
Cannot show the node usage data most of the time.

## Steps to Reproduce (for bugs)
1. pump a bunch of messages (>500k) into graylog via filebeat input
2. check the node statistics on /system/nodes

## Your Environment
* Graylog Version: graylog-server.noarch-3.0.0-12
* Elasticsearch Version: elasticsearch.noarch-5.6.16-1
* MongoDB Version: mongodb-server.x86_64-2.6.12-6.el7
* Operating System: Linux 3.10.0-957.10.1.el7.x86_64 #1 SMP Mon Mar 18 15:06:45 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux || CentOS Linux release 7.6.1810 (Core)
* Browser version: Chrome Version 72.0.3626.121 64-bit

**Comment by kroepke** [comment]
The issue to look at here is that the page itself should be viewable and if metrics cannot be loaded due to an overloaded Graylog backend server, the metrics can be missing, but the overall page should not fail.

**Comment by ShiroDN** [comment]
I also encountered the same problem when graylog is under high load.

OS: Debian 9
Graylog Version: 3.0.2-1

**Comment by ion-storm** [comment]
Can confirm this happens, also the node dissapears from the graylog interface briefly during 50k message spikes.

**Comment by bernd** [comment]
This has been fixed in #9358 and will be in the upcoming 3.3.9 and 4.0.0 releases. Thank you for the report!
